include/userobjects/NumberDensityAccumulator.h

```diff
@@ -0,0 +1,34 @@
+//* This file is part of FENIX: Fusion ENergy Integrated multiphys-X,
+//* A multiphysics application for modeling plasma facing components
+//* https://github.com/idaholab/fenix
+//* https://mooseframework.inl.gov/fenix
+//*
+//* FENIX is powered by the MOOSE Framework
+//* https://www.mooseframework.inl.gov
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+//*
+//* Copyright 2024, Battelle Energy Alliance, LLC
+//* ALL RIGHTS RESERVED
+//*
+
+#pragma once
+#include "ParticleQuantityResidualAccumulatorBase.h"
+// including ray for RayDataIndex
+#include "Ray.h"
+class PICStudyBase;
```

Comment:
```suggestion
```

Is this still necessary to include? I am comparing it to `ChargeDensityAccumulator`, which has similar header structure.

Reply:
No it should be good to remove since the base class contains this. Good catch!